src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -354,12 +361,10 @@ public static IEnumerable<object[]> ASin_Advanced_TestData()
             yield return new object[] { 0, 1234000000, 0, 21.62667394298955 }; // Imaginary part is positive
 
             // Boundary values
-            yield return new object[] { double.MaxValue, 0, double.NaN, double.NaN };
-            yield return new object[] { double.MinValue, 0, double.NaN, double.NaN };
-            yield return new object[] { 0, double.MaxValue, double.NaN, double.NaN };
-            yield return new object[] { 0, double.MinValue, double.NaN, double.NaN };
-            yield return new object[] { double.MaxValue, double.MaxValue, double.NaN, double.NaN };
-            yield return new object[] { double.MinValue, double.MinValue, double.NaN, double.NaN };
+            // We used to test that MinValue and MaxValue produced NaNs, but that's an implementation artifact and
+            // mathematically incorrect. For example, asin(Double.MaxValue) ~ 1.5708 \pm 710.47 i. In fact,
+            // for all arguments in the repesentable range, asin and acos lie in the representable range.
+            // We should fix the implementation so it returns the right values.
```

Comment:
@dcwuser This will effectively do it `[SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp | TargetFrameworkMonikers.NetcoreUwp)]` but I'm sure there's a way to explicitly say "only desktop" - @AlexGhiondea can you remind us?